package.json

```diff
@@ -4,6 +4,7 @@
     "format": "prettier --write .",
     "prepare": "husky"
   },
+  "bin": "./bin/cli.mjs",
```

Comment:
This package has no name, so how we invoke this binary with npx?

Reply:
should we use an npm scope for our packages, for example `@node-website`?

Reply:
For packages "intended to be used only by Node.js contributors" we have `@node-core`.
See https://github.com/nodejs/TSC/issues/1178.

Also these proposals for packages "intended to be used directly by general developers":
- https://github.com/nodejs/admin/issues/794
- https://github.com/nodejs/admin/pull/795

Reply:
This is indeed intended to be used just by contributors.

Reply:
I added the package name as `@node-core/api-docs-tooling`, you can use it in CLI as `npx api-docs-tooling -h`